modules/ROOT/pages/_partials/gs-adding-custom-3scale-routes.adoc

```diff
@@ -6,7 +6,10 @@ endif::[]
 
 = Creating custom URLs for 3scale APIs
 
-NOTE: You cannot delete any routes created using the procedure described below.
+[NOTE]
```

Comment:
@joaedwar just curious, why the different syntax?

Reply:
@finp I believe it renders slightly differently (than when you use NOTE:). I noticed two instances of it when I started the peer review so just ran with it.  What do you prefer?